rules/windows/execution_suspicious_psexesvc.toml

```
@@ -17,14 +17,45 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious Process Execution via Renamed PsExec Executable"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating PsExec Network Connection
+
+PsExec is a remote administration tool that enables the execution of commands with both regular and SYSTEM privileges on Windows systems. It operates by executing a service component, "Psexecsvc", on the remote system, which then runs the specified process and returns the results to the local system. Microsoft develops PsExec as part of the Sysinternals Suite. Although commonly used by administrators, PsExec is frequently used by attackers to enable lateral movement and execute commands as SYSTEM to disable defenses and bypass security protections.
+
+This rule identifies instances where the PsExec service component is executed using a custom name, which can indicate an attempt to bypass security controls or detections that look for its default name.
+
+#### Possible investigation steps
+
+- Check if the usage of this tool complies with the organization's administration policy.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Identify the target computer and its role in the IT environment.
+- Investigate what commands were run, and assess whether this behavior is prevalent in the environment by looking for similar occurrences across hosts.
+
+### False positive analysis
+
+- This mechanism can be used legitimately. As long as the analyst did not identify suspicious activity related to the user or involved hosts, and the tool is allowed by the organization's policy, such alerts can be dismissed.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+  - Prioritize accordingly with the role of the servers and users involved.
```

I'm not quite sure what's being suggested here. Are we recommending to prioritize cases involving servers and users? Or are we suggesting to prioritize based on the importance of certain server or user roles? Or are we suggesting something else entirely?

---

Yeah, to prioritize based on the how critical/important they are

---

Maybe something like this?
```suggestion
  - Prioritize cases involving critical servers and users.
```